roles/sfia/delivery_manager.md

```diff
@@ -0,0 +1,89 @@
+
+Builds strong relationships with customers and ensures customer satisfaction - proven focus on user-based outcomes when influencing ways of working and delivery outcomes
+
+Able to facilitate discussions including technical and business stakeholders and help arrive to helpful conclusions. 
+
+Able to manage stakeholders’ expectations and be flexible, regularly demonstrating proactive rathen than reactive communication. Facilitates difficult discussions within the team or with diverse senior stakeholders.
+
+Guides teams to focus on the outcomes rather than the process / outputs. 
+
+Accountable for ensuring the team is always performing to a Made Tech standard, resolving, or escalating issues wherever appropriate
+
+### Complexity
+
+Able to lead deliveries that are not well defined from the start - have led deliveries under different, often challenging circumstances
+
+Has exprience dealing with complex situations and can do so in various contexts (e.g with impact to account viability, drastic changes to team shape, with impact to MT reputation, financial results and profitability)
+
+Has experience dealing with difficult customers and can, with support, pave the way to a successful engagement
+
+Able to set the delivery cadence to a sustainable level - can make adjustments to resolve issues and ensure a high and steady delivery pace
+
+Ensures value for users is always delivered at the account level and is able to remediate this by introducing techniques to surface user needs and measure the success of the delivery against those
+
+Able to identify, assess & mitigate risks effectively - understands when escalation is required
+
+Finds ways to help our customers beyond just shipping software, ensuring Made Tech teams always demonstrate our mission and values
+
+Upskills and helps customers & team members to resolve complex, often out of the ordinary, issues
+
+### Knowledge
+
+Has a wealth of agile & lean knowledge and has applied both fundamental & advanced principles in different scenarios with little support; 
+
+Able to identify and compare the best processes or delivery methods to use. 
+
+Able to recognise when something does not work and encourages a mindset of experimentation and continuous improvement.
+
+Has a wealth of delivery knowledge and experience; will typically have delivered at least a few tech projects with proven user-focused outcomes
+
+Has good knowledge of modern software development/delivery practices, can discuss their respective benefits and principles & has applied these in multiple deliveries, helping teams become high-performing and successfully deliver outcomes
+
+Able to measure and evaluate delivery outcomes
+
+Remains at the forefront of both agile & delivery topics & new ideas, ensuring they gain new knowledge proactively and regularly
+
+Shares this knowledge with the wider organisation
+
+Shares this knowledge with the customers at every opportunity
+
+### Business Skills
```

Comment:
This section has additional extra lines compared to other sections - suggest keeping them consistent